glib2/ext/glib2/rbglib_fileutils.c

```diff
@@ -53,7 +53,7 @@ void        g_dir_close                     (GDir *dir);
 static VALUE
 rbglib_m_format_size_for_display(G_GNUC_UNUSED VALUE self, VALUE size)
 {
-    return CSTR2RVAL_FREE(g_format_size_for_display(NUM2INT(size)));
+    return CSTR2RVAL_FREE(g_format_size_for_display(NUM2ULONG(size)));
```

Comment:
I added this one: https://github.com/ruby-gnome2/ruby-gnome2/pull/414/files#diff-c70a30246e4ebcd1836b052da6f4af02R27.